src/components/Comments.jsx

```diff
@@ -0,0 +1,26 @@
+import { fetchComments } from "../apiCalls/fetchComments";
+import { useState, useEffect } from "react";
+
+export const Comments = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  useEffect(() => {
+    fetchComments(article_id).then((comments) => {
+      setComments(comments);
+    });
+  });
+
```

Comment:
⚠️ this is causing an infinate re-render & api-call loop!

put in a console.log and check your browsers network tab